### PR TITLE
Dec 14 translations

### PR DESCRIPTION
**Pull request by cvvergara**
This has the modifications on existing translated files,
and a couple of scripts that help me check them

### CODE REVIEW

**Comment by sanak** [comment]
Hi, does this PR's ja localization files will not be changed ?
If so, I am thinking to pull ja files to our OmegaT base working repository's `source` folder.
https://github.com/OSGeo-jp/OSGeoLive-doc-omegat

**Comment by cvvergara** [comment]
@sanak 
Basically what this PR is doing is the following
1) Pull from transifex already existing PO files for each language (maybe a translation was fixed)
2) Fore each language I verify that links and sphinx syntax is ok
  2.1) In case it is not ok, I go to transifex and fix the issues I see
  2.2) Once fixed I commit the language

The last one was Japanese, so the last commit message needs to have all languages for the actions to trigger.

I am working on adding the at least 80% translated files, so more Japanese files will be downloaded

By the way, I will use this opportunity on Japanese translation there are a lot of issues, mainly because Japanese do not use spaces and sphinx commands must be separated by a separator.
I know that particles are separators in Japanese, but sphinx does not.
So the "。" it is considered a separator, but the "は" is not
```
。`GeoWebCache <https://www.geowebcache.org>`_は
```
Sphinx creates an error then I go to transifex and change to:
```
。`GeoWebCache <https://www.geowebcache.org>`_ は
```
Other example:
From this output:
```
/home/vicky/OSGeoLive/docs/cvvergara/build/doc/quickstart/mapnik_quickstart.rst:57: ERROR: Inline interpreted text or phrase reference start-string without end-string.
/home/vicky/OSGeoLive/docs/cvvergara/build/doc/quickstart/mapnik_quickstart.rst:57: ERROR: Inline interpreted text or phrase reference start-string without end-string.
looking for now-outdated files... none found
```
This is the corresponding text:
image
So I fix as follows
image
Note the space I have to add.
Now there are no problems with that ERROR.
I must mention that locally I run with a flag that makes a WARNING to be an ERROR, but on the Actions they are marked as WARNING.

It would be great if you communicate this problem to the Japanese translators.
Basically make sure that the sphinx command is surrounded by a separator

**Comment by cvvergara** [comment]
@sanak Example action WARNINGs
https://github.com/cvvergara/OSGeoLive-doc/runs/1565543511?check_suite_focus=true#step:7:347

**Comment by cvvergara** [comment]
Merging as I will make a PR that depends on this one

**Comment by sanak** [comment]
@cvvergara
Okay, thanks for confirmation.
About Japanese warning issue, we noticed that it often happen,
https://github.com/OSGeo-jp/OSGeoLive-doc-omegat/issues/4
so, I am writing check python script for that.
https://github.com/OSGeo-jp/OSGeoLive-doc-omegat/pull/6/files

Anyway, thanks for fixing many Japanese translation issue on the Transifex. :bow:
